src/diamond/handler/pubsub.py

```diff
@@ -0,0 +1,216 @@
+# coding=utf-8
+
+"""
+Send metrics to [Google Pub/Sub](https://cloud.google.com/pubsub/).
+
+#### Copyright
+
+Copyright 2016 Google Inc.
```

Comment:
Because I work for Google so anything I contribute back has to have this included.